v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Parallel Statement Execution
+summary: The parallel statement execution feature allows parallel execution of multiple independent SQL statements within a transaction.
+toc: false
+---
+
+CockroachDB supports parallel execution of independent [`INSERT`](insert.html), [`UPDATE`](update.html), [`UPSERT`](upsert.html), and [`DELETE`](delete.html) statements within a single transaction. Parallel execution of independent SQL statements reduces the overall execution time of multiple SQL statements, thereby improving query performance.
+
+<div id="toc"></div>
+
+## Why Use Parallel Statement Execution
+
+Applications usually issue multiple SQL statements within a single transaction. SQL engines normally execute each SQL statement sequentially. Because the queries are executed locally, the results can be delivered fast. But CockroachDB being a distributed database, execution of queries takes longer than normal SQL engines because of high round-trip latencies required for consensus. To counteract the issue and optimize query performance, CockroachDB supports parallel execution of independent SQL statements. Thus, despite the higher latency of single data-modifying statements, the overall latency of multiple such statements can be reduced.
```

Comment:
my other point was "can take longer" instead of "take longer". We don't need to be overly pessimistic :)